apis/projects/v1alpha1/types.go

```diff
@@ -0,0 +1,206 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+)
+
+// ProjectParameters define the desired state of an ArgoCD Git Project
+type ProjectParameters struct {
+	// SourceRepos contains list of repository URLs which can be used for deployment
+	// +optional
+	SourceRepos []string `json:"sourceRepos,omitempty"`
```

Comment:
can we add `SourceReposRef` and `SourceReposSelector` here and generate them via https://github.com/crossplane/crossplane-tools#reference-resolvers?